package.json

```diff
@@ -29,9 +29,10 @@
     "susy": "^2.2.9"
   },
   "devDependencies": {
+    "backbone": "*",
     "browser-sync": "*",
     "del": "*",
-    "edx-ui-toolkit": "~0.3.0",
+    "edx-ui-toolkit": "https://github.com/edx/edx-ui-toolkit.git#4bc0a3ac2ef92f5ee044aaab9230826542798979",
```

Comment:
Just a reminder not to merge until we've done a UI Toolkit release and this can refer to a version number.